doc/source/dev/contributor/rendering_documentation.rst

```diff
@@ -21,9 +21,9 @@ Rendering Documentation Locally
 To render the documentation on your own machine (macOS or Linux):
 
 0. Ensure that you have a working SciPy :ref:`dev-env` active.
-#. Install `Sphinx`_ or ensure that your installation is up to date. For
+#. Install `Sphinx`_ and `matplotlib`_. For
    example, if you're using the Anaconda distribution of Python, enter in a
-   terminal window ``conda install sphinx`` or ``conda update sphinx``.
+   terminal window ``conda install sphinx matplotlib``.
 #. In a terminal window, browse to the ``scipy/doc`` directory. Note the
```

Comment:
As already discussed I think `python setup.py build_sphinx` is the recommended way of building doc.

Reply:
I still need to try that in the container.
I did try it on my Mac, but it failed.

Reply:
And in the docker container it doesn't build the SciPy documentation, it builds some sort of `scipy-sphinx-theme` documentation:
![image](https://user-images.githubusercontent.com/6570539/63551820-f8f97480-c4ea-11e9-9f9d-235e0d523eea.png)

Reply:
The instructions [here](https://14871-1460385-gh.circle-artifacts.com/0/html-scipyorg/dev/contributor/rendering_documentation.html#rendering-documentation) work fine as long as I set up the development environment using Python 3.6. Alternatively, if I manually specify `PYVER:=3.7` in the Makefile, rendering the documentation with Python 3.7 works fine. 

Do you develop with Python 3.6? I'm guess that new developers who need this guide will use 3.7, so could we remove 3.6 from the container? I could add instructions on changing the Makefile; but that might not be the best approach either.

Reply:
I develop with 3.7. There's no point removing 3.6 because that's the default Python for Ubuntu:bionic.

I've just build the documentation using `python setup.py build_sphinx` in docker. The documentation show up in `scipy/build/sphinx/html/`. The index.html looks a little funny though:
![Untitled](https://user-images.githubusercontent.com/702934/63561363-096a2780-c59d-11e9-9278-cbf689763766.png)

Reply:
Take a look at whether it produced output in html-scipyorg.

Reply:
I couldn't find an html-scipyorg folder.